package-lock.json

```diff
@@ -89,8 +89,8 @@
     },
     "node_modules/@braintree/sanitize-url": {
       "version": "2.1.0",
```

Comment:
Version is still referring to 2.1.0. It should be 6.0.0

Reply:
Thanks for catching this, I've made the change, my apologies for the oversight! @pkfec

Reply:
@cnlucas Looks good!